src/synapse/azext_synapse/_help.py

```diff
@@ -122,7 +130,7 @@
 examples:

```

Comment:
This command should suppport updating node count, auto-scale status and so on.

Reply:
> This command should suppport updating node count, auto-scale status and so on.

updated